File: api-gin/src/routes/cliente-routes.go
package routes

import (
	"github.com/gin-gonic/gin"
	controller "tcc-api-gin/src/controllers"
)

func ClienteRoutes(router *gin.RouterGroup) {
	cliente := router.Group("/clientes")
	{
		cliente.POST("/", controller.CreateCliente)
		cliente.POST("/login", controller.Login)
	}
}

File: api-express/src/routes/cliente-routes.js
import { Router } from "express";
import ClienteController from "../controllers/cliente-controller.js";
import PedidoController from "../controllers/pedido-controller.js";
import isAuthenticated from "../middlewares/is-authenticated.js";

const clienteRoutes = Router();

clienteRoutes.post('/', ClienteController.create);
clienteRoutes.post('/login', ClienteController.login);

export default clienteRoutes;